twitter/README.md

```diff
@@ -0,0 +1,12 @@
+# Setting up Twitter API credentials
```

Comment:
It would be nice to add the description you added on the PR in this README, this explains how to set up the plugin, but not what it does.